packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -707,6 +777,21 @@ function IntroductoryOfferCallout( {
 	return <DiscountCallout>{ text }</DiscountCallout>;
 }
 
+function PartnerLogo( { className }: { className?: string } ): JSX.Element | null {
+	const translate = useTranslate();
+
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+	return (
+		<LineItemMeta className={ joinClasses( [ className, 'jetpack-partner-logo' ] ) }>
+			<div>{ translate( 'Included in your IONOS plan' ) }</div>
+			<div className={ 'checkout-line-item__partner-logo-image' }>
+				<img src={ '/calypso/images/jetpack/partners/partner-logo-ionos.png' } alt="IONOS Logo" />
```

Comment:
I know this may still be in-progress, but this URL could potentially be a problem if this component was ever rendered by a package outside of calypso (eg: `mini-cart` when it's [used in wp-admin](https://github.com/Automattic/wp-calypso/pull/52890), even though it currently doesn't import `WPCouponLineItem`).

However, it's not simple to include an image like this inside an npm package, so my suggestions are either:

- Move `PartnerLogo` and `WPCouponLineItem` into calypso (the `client` directory). This is probably the easiest thing unless you think this should be included in `mini-cart`.
- Make the image an SVG and move it to the `wpcom-checkout` package, like [some of our payment logos](https://github.com/Automattic/wp-calypso/blob/8b48c93f4f9732481d87e24dd1fd819bc260fd25/packages/wpcom-checkout/src/google-pay-mark.tsx). This is much less efficient but would work.

Reply:
Hey @sirbrillig , thanks for the suggestions on this. I still don't know much about how and where the `wpcom-checkout` package is being used so I greatly appreciate your effort on explaining your suggestions.

I've converted the IONOS logo image to an SVG and created an SVG component like you suggested. We should serve different partner logos through and API if we decide to replicate this strategy for other partners, if so we will update this for it to work with such API, which will then eliminate the need of storing a partner logo as an SVG on the code base.